### DISCUSSION TITLE
Fine tuning of Frigate+ model

### DISCUSSION THREAD

**Discussion by osemmler** [blakeblackshear/frigate#9943] [Ask A Question]
### What is your question:

I would like to ask if it is appropriate to change the camera parameters during image acquisition for training the Frigate+ model?

Specifically, this is a minor adjustment to the following parameters:
- direction of view
- rotation of the view
- brightness

**Would you recommend removing the original images or is it instead advantageous to vary the data (different brightness and rotation of objects)?**

___

Furthermore, I didn't find any tips on how to improve the results of my own Frigate+ model (except this page https://docs.frigate.video/plus/improving_model). If I just searched wrong, I apologize and ask for a link. If a similar page does not exist I suggest you create one.

**Comment by NickM-27** [comment]
in my experience having moved a couple cameras in the way that you are describing I have not seen any issues with my model

**Comment by blakeblackshear** [comment]
It should be fine. Even with angle changes or different settings, your camera images still similar relative to many other users.

That guide in the docs is the only guide for improving your model.

**Comment by osemmler** [comment]
Thank you for your answers.

I also have a question about "baseline model" from https://docs.frigate.video/plus/:

If I generate a model in Frigate+ is it generated from:
1. **"baseline model" + my current set of verified images**
2. **my previous model + my current set of verified images**

Which is correct 1 or 2?

My point is that if, for example, I identify errors in my previous images, I can remove them and these erroneous images will not affect the newly generated model in any way.

**Reply by NickM-27** [reply]
See https://docs.frigate.video/plus/faq#are-my-models-trained-just-on-my-image-uploads-how-are-they-built

Each time you request a model the images that are currently in your list will be freshly used to tune on top of the baseline model
